virt/virtual_machines/virt-triggering-vm-failover-resolving-failed-node.adoc

```diff
@@ -0,0 +1,30 @@
+[id="virt-triggering-vm-failover-resolving-failed-node"]
+= Triggering virtual machine failover by resolving a failed node
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-triggering-vm-failover-resolving-failed-node
+toc::[]
+
+If a node fails and xref:../../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-about_deploying-machine-health-checks[machine health checks] are not deployed on your cluster, virtual machines (VMs) with `RunStrategy: Always` configured are not automatically relocated to healthy nodes. To trigger VM failover, you must manually delete the `Node` object.
```

Comment:
It would be good if we could document why it is available with IPI deployment but not with UPI deployment. It seems the confusion that we saw came from this difference between both installation scenario.

Reply:
I don't fully understand why it's supported with IPI but not UPI, because I only know what Dan said in the BZ ("Supporting node recycling on UPI would be a major undertaking for OCP.")

@dankenigsberg is there a specific message I should convey here?

Reply:
Unfortunately I don't know much more than what you've written.. if I am not mistaken, UPI lack the Machine entity which MHC depends on.

I suspect @beekhof  can provide an auhoritative background story. But his bottom line would be the same: if you care to keep your VM running you need to have ipi and enable mhc.

Reply:
Thanks for weighing in, Dan. The bottom line is the most important part, I think. I am going to merge the doc as-is, but if there is additional info that would help customers, I can add it later.